include/avif/avif.h

```diff
@@ -749,6 +749,13 @@ AVIF_API void avifImageStealPlanes(avifImage * dstImage, avifImage * srcImage, a
 // the number of threads that can ever be in flight at a given time, please account for this
 // accordingly.
 
+// ---------------------------------------------------------------------------
+// Scaling
+
+// Scales the YUV/A planes in-place. dstWidth and dstHeight must both be <= AVIF_DEFAULT_IMAGE_DIMENSION_LIMIT and
+// dstWidth*dstHeight should be <= AVIF_DEFAULT_IMAGE_SIZE_LIMIT.
```

Comment:
I wonder if we can figure out the image dimension and the image size limits of libyuv and use those limits instead.

Reply:
existing code uses libavif's internal limits even though it calls libyuv. so i think it is fine.